llama_stack/providers/inline/datasetio/localfs/datasetio.py

```diff
@@ -49,15 +54,27 @@ async def load(self) -> None:
             raise ValueError(f"Unsupported dataset source type: {self.dataset_def.source.type}")
 
         if self.df is None:
-            raise ValueError(f"Failed to load dataset from {self.dataset_def.url}")
+            if self.dataset_def.source.type == "uri":
+                raise ValueError(f"Failed to load dataset from {self.dataset_def.source.uri}")
+            else:
+                raise ValueError("Failed to load dataset from rows")
+
+
+class LocalFSDatasetStore(DatasetStore):
```

Comment:
not addressed https://github.com/meta-llama/llama-stack/pull/2662#discussion_r2195269817